app/parse.py

```diff
@@ -15,8 +31,75 @@ class Product:
     num_of_reviews: int
 
 
+QUOTE_FIELDS = [field.name for field in fields(Product)]
+
+
+_driver: WebDriver | None = None
```

Comment:
The global variable `_driver` is initialized as `None`, but it is used in the `get_driver` function before being set by `set_driver`. Ensure that `set_driver` is called before any function that relies on `_driver` to avoid potential `NoneType` errors.